lib/jormungandr/test/integration/Test/Integration/Jormungandr/Scenario/API/StakePools.hs

```diff
@@ -99,3 +119,195 @@ spec = do
             r <- request @ApiStakePool ctx (method, "v2/stake-pools") Default Empty
             expectResponseCode @IO HTTP.status405 r
             expectErrorMessage errMsg405 r
+
+    it "STAKE_POOLS_JOIN_01 - Can join a stakepool" $ \ctx -> do
+        (_, p:_) <- eventually $
+            unsafeRequest @[ApiStakePool] ctx listStakePoolsEp Empty
+        w <- emptyWallet ctx
+        r <- joinStakePool ctx p (w, "Secure Passphrase")
+        expectResponseCode HTTP.status501 r
+
+    it "STAKE_POOLS_JOIN_01 - I cannot join another until I quit\
+        \ or maybe I will just re-join another?" $ \ctx -> do
+        (_, p1:p2:_) <- eventually $
+            unsafeRequest @[ApiStakePool] ctx listStakePoolsEp Empty
+        w <- emptyWallet ctx
+        r1 <- joinStakePool ctx p1 (w, "Secure Passprase")
+        expectResponseCode HTTP.status501 r1
+
+        r2 <- joinStakePool ctx p2 (w, "Secure Passprase")
+        expectResponseCode HTTP.status501 r2
+
+    it "STAKE_POOLS_JOIN_01 - I definitely can quit and join another" $ \ctx -> do
```

Comment:
This line is only _slightly_ over the [line length limit](https://github.com/input-output-hk/cardano-wallet/wiki/Coding-Standards#limit-line-length-to-80-characters), but I hope you can forgive me for using it to illustrate a point. :smile: 

Of course we want to make the description long enough that the test is self-documenting. However, the line length limit seemingly imposes a constraint on the description length.

Though perhaps there's a way to avoid the above issue. How about using a format similar to the following:
 
```hs
    it "STAKE_POOLS_JOIN_01: \
        \I definitely can quit and join another."
        $ \ctx -> do
            (_, p1:p2:_) <- eventually $
                unsafeRequest @[ApiStakePool] ctx listStakePoolsEp Empty
            w <- emptyWallet ctx
            ...
```
This format has the advantage of scaling nicely even when the description gets longer:

```hs
    it "LAUNCH_MISSILES_01: \
        \When initiating a missile attack, rockets are fired in such a way \
        \as to prevent excessive collateral damage to neighbouring friendly \
        \countries."
        $ \ctx -> do
            (_, p1:p2:_) <- eventually $
                unsafeRequest @[ApiStakePool] ctx listStakePoolsEp Empty
            w <- emptyWallet ctx
            ...
```
@piotr-iohk What do you think?

Reply:
@jonathanknowles agreed and fixed. :)